discord/guild.py

```diff
@@ -1169,6 +1170,51 @@ def _create_channel(self, name, overwrites, channel_type, category=None, **optio
 
         return result
 
+    async def fetch_all_custom_emojis(self):
```

Comment:
Nowhere in the library is the term `fetch_all` used. Instead, normal English plurals are used.